management/us-east-1/security-hub/security_hub.tf

```diff
@@ -0,0 +1,24 @@
+# Multi account with organizations
+## Enable the security standards that Security Hub has designated as default:
+## AWS Foundational Security Best Practices v1.0.0 and CIS AWS Foundations Benchmark v1.2.0
+resource "aws_securityhub_organization_admin_account" "main" {
+  depends_on = [aws_organizations_organization.main]
+
+  admin_account_id = var.accounts.security.id
+}
+
+resource "aws_securityhub_finding_aggregator" "main" {
+  linking_mode = "ALL_REGIONS"
+
+  depends_on = [aws_securityhub_organization_admin_account.main]
+}
+
+resource "aws_securityhub_organization_configuration" "main" {
+  auto_enable           = true
+  auto_enable_standards = "DEFALT"
```

Comment:
Should be DEFAULT?

Reply:
@martingaleano as per Terraform docs:

> Whether to automatically enable Security Hub default standards for new member accounts in the organization.

So, we set it to `DEFAULT` for new members to have enabled these defaults.

The other option is NONE:

> To opt out of enabling default standards for new member accounts, set this parameter equal to NONE

Reply:
I was asking because it says `DEFALT` instead of `DEFAULT`. Typo?  hehe

Reply:
FYI Github provides this "Add suggestion" feature which is very helpful for cases like this.
<img width="103" alt="Screen Shot 2024-05-31 at 16 18 27" src="https://github.com/binbashar/le-tf-infra-aws/assets/38356409/deda0a93-1cfe-442b-ac35-ee4301aa28cd">